### ISSUE TITLE
Status expressed via title element compliant with SC 4.1.3?

### ISSUE THREAD

**Issue by brennanyoung** [w3c/wcag#2258]
This might be a bit of a corner case, but seems worth mentioning.

[The HTML5 spec](https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element) for the `application-name` attribute in `<meta>` touches upon the real world, common use of `<title>` elements to convey **status**. The relevant copy goes like this:

> User agents may use the application name in UI in preference to the page's [title](https://html.spec.whatwg.org/multipage/semantics.html#the-title-element), since the title **might include status messages** and the like relevant to the status of the page at a particular moment in time instead of just being the name of the application.

(emphasis mine)

It might be argued that using `<title>` for status messages is "a bit of a hack", and there will be accessibility issues if relying on this mechanism alone, especially in relation to WCAG SC 4.1.3: Status Messages. 

It might also be argued that this practice is common enough that it **should** be an accepted way of communicating status - after all, the title **is** communicated to ATs programmatically. It's just not a live region. (Maybe it should be?) The [ARIA in HTML spec](https://html.spec.whatwg.org/multipage/semantics.html#the-title-element) says that `<title>` takes no role or aria attributes.

However I don't think we can expect the title to be re-announced when it changes with current implementations. (Accessibly Supported), so I would shy away from relying on title alone to comply with SC 4.1.3

Forthcoming HTML5 features for informing ATs about client side rerouting seems highly relevant here, since a re-render is effectively a change of page status.

Any opinions on this? 
- Does it deserve a note and/or Failure example in the "Understanding SC 4.13" doc?
- Is it worth lobbying the whatwg folks to tidy up this potentially anti-a11y loophole in their spec? (https://github.com/whatwg/html/issues/7701)
- Should we go along with the real world, and imagine title as a potential bonafide channel for status messages?
- Do we need to raise the point anywhere in the WCAG spec or other docs that `<meta application-name>` is intended for a static page title, whereas `<title>` might (according to the spec) contain status messages?
- Maybe the AOM notifications API folks have input?

**Comment by JAWS-test** [comment]
> However I don't think we can expect the title to be re-announced when it changes with current implementations.

If JavaScript is used to update the page title, then this is not automatically output by a screen reader. To make matters worse, the screen reader does not notice the change, i.e. when the page title is forced to be output again, the old one is still output and not the new one. At least this is the case in Firefox, Chrome and Edge (with JAWS and NVDA).

**Comment by patrickhlauke** [comment]
> Should we go along with the real world, and imagine title as a potential bonafide channel for status messages?

i'd say we can't, because it objectively doesn't do what the SC intends (that it's done in a way that is noticed by AT, in their current accessibility-supported form at least)

so i'd say a failure technique that shows this, and suggest that a change in title should be doubled-up with a live region notification, might be in order

**Comment by brennanyoung** [comment]
If there is a consensus for adding a Failure Technique for SC 4.1.3, we can close this, and keep https://github.com/w3c/wcag/issues/2261

**Comment by alastc** [comment]
I don't know if there is consensus for that, but it seems like the newer issue would be the one to discuss & action, so I'll close this one. (Feel free to re-open if I've missed something and there is a different topic to progress here.)
